README.md

```diff
@@ -1 +1,36 @@
-# syncer
\ No newline at end of file
+# action-sync-fork
+
+Syncs galaxy integration forks with upstream
+
+## Example Usage (workflow) - OUTDATED
+
+[`.github/workflows/sync.yml`](https://github.com/GOG-Nebula/galaxy-riot-integration/blob/9da15bdfd5f30f2aa6e887491d7f53d0853adeae/.github/workflows/sync.yml)
+
+```yaml
+name: sync-upstream
+on:
+  schedule:
+    - cron: '23 * * * *'
+  workflow_dispatch:
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GOG-Nebula/action-sync-fork@feature/initial
```

Comment:
do not forget to switch branch to `main` before merge

Reply:
Oh yeah thanks!
```suggestion
      - uses: GOG-Nebula/action-sync-fork@main
```